src/SDKs/DataFactory/Management.DataFactory/changelog.md

```diff
@@ -4,6 +4,14 @@
 
 ### Feature Additions
   * Cancel pipeline run api.
+  * Add AzureMySql linked service.
```

Comment:
You will need to update the version for these changes in your csproj file (0.3.0-preview or 0.2.1-preview).

Reply:
We'll be bumping the version in a separate change soon (to 0.2.1-preview), thanks for the reminder